public/js/pages/localweb.js

```diff
@@ -20,9 +20,35 @@ requirejs.config({
   }
 });
 
-require(["jquery", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, languages, selectize ) {
+require(["jquery", "analytics", "languages", "selectize", "transition", "collapse", "carousel", "magnific-popup"], function ($, analytics, languages, selectize ) {
   "use strict";
   var $window = $(window);
+  var TIME_DELAYED_REDIRECT = 1000; // in milliseconds
+  // analytics
+  var delayRedirect = function(url) {
+    setTimeout(function() {
+      window.location.href = url;
+    }, TIME_DELAYED_REDIRECT);
+  };
+  $("#header-partner-with-us-link").click(function() {
+    analytics.event("Click on Partner with Us");
+  });
+  $("#try-the-open-beta-link").click(function() {
+    analytics.event("Try the Open Beta");
+  });
+  $("#moi-video-play-btn").click(function() {
+    analytics.event("Play Video", {label: "MOI Partner Opportunity"});
+  });
+  $(".report-links").click(function(event) {
+    event.preventDefault();
+    analytics.event("Download Report", {label: $(this).data("report-name")});
+    delayRedirect($(this).attr("href"));
+  });
+  $("#mobile-webmaker-partners-link").click(function() {
```

Comment:
This ID was added to the locale source file few hours ago (see [here](https://github.com/mmmavis/webmaker.org/blob/f0d46e07a6149651b04cddc1ce8a14826d89ad83/locale/en_US/localweb.json#L83)) It probably hasn't got uploaded to s3 yet.